### ISSUE TITLE
Wrong number for visibility

### ISSUE THREAD

**Issue by E3V3A** [supercoderz/pyflightdata#7]
Checking the `visibility` item for in the `api.get_airport_weather`, we get some funny results. Here is one: 

```python
>>> blist = api.get_airport_weather("TEX");
>>> blist
{'metar': 'KTEX 182115Z AUTO 27007KT 10SM CLR 08/M13 A3027 RMK AO2 T00771128', 
'time': '1516310100', 'qnh': '1025', 'dewpoint': {'celsius': '-13', 'fahrenheit': '9'}, 'humidity': '22', 'pressure': {'hg': '30', 'hpa': '1025'}, 'sky': {'condition': {'text': 'Clear'}, 
'visibility': {'km': '16093000', 'mi': '10', 'nmi': '9'}}, 
'flight': {'category': 'None'}, 'wind': {'direction': {'degree': '270', 'text': 'From west'}, 'speed': {'kmh': '14', 'kts': '8', 'mph': '9', 'text': 'Mod'}}, 'temp': {'celsius': '8', 'fahrenheit': '46'}, 
'elevation': {'m': '2767', 'ft': '9078'}, 'cached': '1516311196'}
```

The visibility here is `16093000` Km...

**Comment by hari-allamraju** [comment]
This appears to be the case for other airports as well; could be fr24 way of indicating invalid data? Will investigate

**Comment by hari-allamraju** [comment]
It looks like an issue with conversion - 10mi is approx 16.093 KM - so somehow the units are messed up - can fix with adding a (reluctant) conversion

**Comment by hari-allamraju** [comment]
Fixed in latest version 0.5.6
